packages/common/sagas/entities/item.js

```diff
@@ -484,7 +496,7 @@ export function* createItemSaga({
       yield put(addItemToList(newItem));
     }
 
-    if (teamId) {
+    if (teamId && !isSystemItem) {
```

Comment:
We need to remove the logic of sharing the team item to members of the team.

Reply:
Done